hvac/api/secrets_engines/transit.py

```diff
@@ -335,7 +335,7 @@ def encrypt_data(
 
         :param name: Specifies the name of the encryption key to encrypt against. This is specified as part of the URL.
         :type name: str | unicode
-        :param plaintext: Specifies base64 encoded plaintext to be encoded.
+        :param plaintext: Specifies base64 encoded plaintext to be encoded. Not needed if batch_input is set.
```

Comment:
```suggestion
        :param plaintext: Specifies base64 encoded plaintext to be encoded. Ignored if ``batch_input`` is set, otherwise required.
```